components/automate-gateway/api/compliance/reporting/stats/stats.proto

```diff
@@ -77,47 +106,73 @@ service StatsService {
 message Empty {}
 
 message ListFilter {
+	// The list of values to filter on for the given type. We 'OR' between these fields.
 	repeated string values = 20;
+	// The field to filter on.
 	string type = 21;
 }
 
 message Query {
+	// Unique idenfifier, such as a profile id.
 	string id = 17;
+	// Type of data being requested, used for ReadTrend and ReadSummary.
 	string type = 19;
+	// The number of results to return (used when pagination is not supported).
 	int32 size = 18;
+	// The interval to use for ReadTrend results, in integer seconds. Default of one hour, 3600.
 	int32 interval = 25;
+	// Filters applied to the results.
 	repeated ListFilter filters = 20;
+	// Sort the results in ascending or descending order.
 	enum OrderType {
 		ASC = 0;
 		DESC = 1;
 	}
 	OrderType order = 21;
+	// Sort the list of results by a field.
 	string sort = 22;
+	// The offset for paginating requests. An offset defines a place in the results in order to fetch the next page of the results.
 	int32 page = 23;
+	// The number of results on each paginated request page.
 	int32 per_page = 24;
 }
 
 message Summary {
+	// Statistics for the controls executed in the compliance reports.
 	ControlsSummary controls_summary = 1;
+	// Statistics about the nodes scanned in the compliance reports.
 	NodeSummary node_summary = 2;
+	// Statistics on the overall compliance reports.
 	ReportSummary report_summary = 3;
```

Comment:
I haven't checked this locally, but I think these probably won't show up in the browser. The ones commented `// Intentionally blank.` in `reporting.proto` have the same behavior due to a limitation of swagger. The description for a single message reference is taken from where the message is defined, not where its referenced. That is, you'd need to document this line to provide documentation for this field: https://github.com/chef/automate/pull/2054/files#diff-35e6bfeb760490166078b62bdee851c4R196

Reply:
oooooh interesting!

Reply:
oooh i see. if it's a repeated field, the description written above the field in the message is included. if it's not a repeated field, we end up in this situation. got it.